server/metrics/fdb.go

```diff
@@ -42,31 +45,51 @@ var (
 	}
 )
 
-func GetFdbReqTags(reqMethodName string, tx bool) map[string]string {
+func getFdbReqTags(reqMethodName string, tx bool) map[string]string {
 	return map[string]string{
-		"method": reqMethodName,
-		"tx":     strconv.FormatBool(tx),
+		"method":    reqMethodName,
+		"tx":        strconv.FormatBool(tx),
+		"namespace": DefaultReportedNameSpace,
 	}
 }
 
-func GetFdbReqSpecificErrorTags(reqMethodName string, code string, tx bool) map[string]string {
+func getFdbReqSpecificErrorTags(reqMethodName string, code string, tx bool) map[string]string {
 	return map[string]string{
 		"method":     reqMethodName,
 		"error_code": code,
 		"tx":         strconv.FormatBool(tx),
+		"namespace":  DefaultReportedNameSpace,
 	}
 }
 
+func addNameSpaceToTags(ctx context.Context, tags map[string]string) map[string]string {
+	namespace, err := request.GetNamespace(ctx)
+	if err != nil && err.Error() != "Namespace not found" {
```

Comment:
can we construct a proper error when `namespace not found` and then use that for comparison here?

Reply:
This is done.